projects/developer/src/app/common/pipes/truncate.pipe.ts

```diff
@@ -0,0 +1,13 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+    name: 'truncate',
+})
+export class TruncatePipe implements PipeTransform {
+    transform(value: string, limit = 25, completeWords = false, ellipsis = '...') {
```

Comment:
Can we just use the lodash version?

Reply:
Replaced truncate pipe logic with lodash.